src/main/resources/templates/laDigitalAssister/jobPayAmount.html

```diff
@@ -10,21 +11,27 @@
         <div th:replace="~{fragments/goBack :: goBackLink}"></div>
         <main id="content" role="main" class="form-card spacing-above-35">
           <p class="grey-text" th:text="${fieldData.get('employerName')}"></p>
+          <th:block th:with="payPeriod =${fieldData.get('payPeriod')}">
+            <th:block
```

Comment:
seems like these blocks can be merged?

Reply:
I believe that's current behavior (just tested it in staging).